src/app/test/mocked-objects.ts

```diff
@@ -456,3 +456,87 @@ export const testSourceFiles: Array<SourceFile> = [
     }
   }
 ];
+
+export const elasticSearchResponse = [
```

Comment:
Even though I worked with you on this and suggested this, so it's my fault, but I think your mock response should also contain a workflow

Reply:
Included a workflow in the mocked-objects